### PR TITLE
Fabo/lunie 3 extension

### PR DESCRIPTION
**Pull request by faboweb**
Adjustments to make the extension Lunie 3 compatible

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5fb70efa5c170a0090f56d64--lunieio.netlify.app

File: api/lib/reducers/emoneyV0-reducers.js [hunk 1]
```diff
@@ -35,5 +35,6 @@ module.exports = {
   blockReducer: cosmosV3Reducers.blockReducer,
   setTransactionSuccess: cosmosV3Reducers.setTransactionSuccess,
   accountInfoReducer: cosmosV3Reducers.accountInfoReducer,
+  delegationReducer: cosmosV3Reducers.delegationReducer,
```
Comment (faboweb) [inline api/lib/reducers/emoneyV0-reducers.js:38]:
```suggestion
```

File: extension/networks.js [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
+      }
+    ],
+    address_prefix: 'cosmos',
+    network_type: 'cosmos',
+    HDPaths: `[{"value":"m/44'/118'/0'/0/0", "name":"Cosmos HD Path"}]`,
+    curves: `[{"value":"ed25519", "name":"Edwards curve"}]`,
+    HDPath: `m/44'/118'/0'/0/0`,
+    curve: 'ed25519'
+  }
+]
```
Comment (Bitcoinera) [inline extension/networks.js:44]: ? Thought we wanted to have all the networks hardcoded for extension?

**Comment by Bitcoinera** [comment]
OK, I'm going to try this for normal/old Lunie

File: extension/networks.js (hunk 2 above)
Comment (faboweb) [reply extension/networks.js:44]: we do, needs a merge from master

Comment (faboweb) [reply extension/networks.js]: done

**Review by Bitcoinera — APPROVED** [review]

**Pull request merged**